### PR TITLE
Add image registry team to pkg/image OWNERS

### PR DESCRIPTION
**Pull request by dmage**
Adding the team that owns the image API to OWNERS.

### CODE REVIEW

**Comment by adambkaplan** [comment]
/hold

@deads2k @soltysh `e2e-cmd` seems to be spinning, with several having errors:

```
Jul 21 06:59:18.661: INFO: pod "test-cmd" errored trying to run the command: <nil>
Jul 21 07:01:02.834: INFO: Logs from the container: Error from server (AlreadyExists): project.project.openshift.io "cmd-debug" already exists
```

(error also applies to build and admin tests)

File: pkg/image/OWNERS [hunk 1]
```diff
@@ -2,14 +2,16 @@ reviewers:
...
   - tnozicka
   - adambkaplan
+  - dmage
+  - ricardomaraschini
 approvers:
   - bparees
   - smarterclayton
   - soltysh
   - mfojtik
   - tnozicka
```
Comment (sttts) [inline pkg/image/OWNERS:15]: @tnozicka @soltysh @mfojtik do we want to give up approvership in this package? 7 approvers are at least 3 or 4 too many IMO.

**Pull request merged**